src/index.js

```diff
@@ -14,12 +14,14 @@ export function useVirtual({
   parentRef,
   horizontal,
   scrollToFn,
+  observer,
```

Comment:
Since this is technically executed as a custom hook, the name should be `useObserver` or something similar.

Reply:
@tannerlinsley Thank you, please review the updated naming